grafana/dashboards/vampire-drain.json

```diff
@@ -50,8 +50,8 @@
       "repeat": "car_id",
       "scopedVars": {
         "car_id": {
-          "selected": true,
-          "text": "1",
+          "selected": false,
+          "text": "Terry",
```

Comment:
It happens very easily that the default values are replaced by the own values. Can you revert this please?

Reply:
Yep, updated with the correct values.